src/backend/scripts/setup_localdata.py

```diff
@@ -94,9 +95,28 @@ def create_location(session, region, country, division, location):
     return location
 
 
-def create_sample(session, group, uploaded_by_user, location, suffix, is_failed=False):
-    private_id = f"{group.prefix}-private_identifier_{suffix}"
-    public_id = f"{group.prefix}-public_identifier_{suffix}"
+def create_pathogens(session):
+    get_or_create = [
+        {"name": "SARS-CoV-2", "slug": "SC2"},
+        {"name": "Monkeypox", "slug": "MPX"},
+    ]
+    pathogens = []
+    for record in get_or_create:
+        pathogen = (
+            session.query(Pathogen).where(Pathogen.slug == record["slug"]).one_or_none()
+        )
+        if not pathogen:
+            pathogen = Pathogen(name=record["name"], slug=record["slug"])
+            session.add(pathogen)
+        pathogens.append(pathogen)
+    return pathogens
+
+
+def create_sample(
+    session, group, pathogen, uploaded_by_user, location, suffix, is_failed=False
+):
+    private_id = f"{group.prefix}-private_identifier_{suffix}_{pathogen.slug}"
+    public_id = f"{group.prefix}-public_identifier_{suffix}_{pathogen.slug}"
```

Comment:
Changes to this script update the local dev environment to add samples/trees/runs to the db for both MPX *and* SC2